frontend/src/components/account/showShareInformationsModal.tsx

```diff
@@ -0,0 +1,92 @@
+import { Text, Divider, Progress, Stack, Group, Flex } from "@mantine/core";
+import { ModalsContextProps } from "@mantine/modals/lib/context";
+import { MyShare } from "../../types/share.type";
+import moment from "moment";
+import { byteToHumanSizeString } from "../../utils/fileSize.util";
+import CopyTextField from "../upload/CopyTextField";
+import { FileMetaData } from "../../types/File.type";
+
+const showShareInformationsModal = (
+  modals: ModalsContextProps,
+  share: MyShare,
+  appUrl: string,
+  maxShareSize: number | string
```

Comment:
```suggestion
  maxShareSize: number
```

`maxShareSize` should not be a string

Reply:
Well, I thought I made that because all the config values were strings, I'll check that